torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1563,16 +1563,27 @@ def constrain_unify(a: torch.SymInt, b: torch.SymInt) -> None:
 # in the unlikely branch.)  (I think expect is a good name; in recent
 # versions of C++, this is replaced with [[likely]], which is weaker
 # and not accurate for this function!)
-def expect_true(a: Union[SymBool, bool], skip: int = 0) -> bool:
+def expect_true(
+    a: Union[SymBool, bool],
+    skip: int = 0,
+    message: Optional[Union[str, Callable[[], str]]] = None,
```

Comment:
can you unify on either message or info? is there a reason they are named differently in different places?